### DISCUSSION TITLE
项目主页

### DISCUSSION THREAD

**Discussion by WSQ0127** [Qexo/Qexo#513] [Q&A] [answered]
大佬的项目主页用的哪个框架和主题啊，好像很多人用

**Comment by Fgaoxing** [comment, marked as answer]
vitepress



在 2024-02-23 18:40:54，"草履虫蒟蒻¿¿¡¡" ***@***.***> 写道：

大佬的项目主页用的哪个框架和主题啊，好像很多人用
